core/app-mgr/app-manager/module_wasm_app.c

```diff
@@ -653,7 +653,8 @@ wasm_app_module_install(request_t * msg)
                 AOT_SECTION_TYPE_FUNCTION,
                 AOT_SECTION_TYPE_EXPORT,
                 AOT_SECTION_TYPE_RELOCATION,
-                AOT_SECTION_TYPE_SIGANATURE
+                AOT_SECTION_TYPE_SIGANATURE,
```

Comment:
I have a concern here:
If the signature section is not the last section, then is may be more difficult to implement the sign/verify logic

Reply:
Here the array defines which sections to be destroyed after module is loaded to save the memory usage. As now signature section isn't generated by wamrc, in aot loader, we checks that the AOT file's last section must be relocation section (when custom section isn't generated) or custom section. When signature section is generated in the future, we can add more strict checks, e.g. the last section must be relocation section, custom section or signature section, and the order must be custom section < relocation section < signature section.